README.md

```diff
@@ -5,28 +5,77 @@
 
 ### What is this repository for? ###
 
-* Address Index is a Play Framework application which resolves address query strings to actual addresses.
-* Version 0.0.1
+Address Index is a Play Framework (2.6) application which matches addresses.
+
+The input can be a complete address (may be poor quality), and the system will use advanced data science techniques to determine the most likely matching AddressBase entries with UPRNs (Unique Property Reference Numbers).
```

Comment:
Poor quality is a bit subjective. How about: 

The input can be a complete or partial address